app/components/orders/company-info.js

```diff
@@ -0,0 +1,7 @@
+import classic from 'ember-classic-decorator';
+import Component from '@ember/component';
+
+@classic
```

Comment:
No classic components

Reply:
i am using classic features here, i should add @classic here

Reply:
Use non classic features

Reply:
sir i removed the property , because banner [Company Information] is required on every page